diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index 273306c8..b3cc6fd0 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -105,12 +105,7 @@ endef
 define Package/uboot-envtools/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/env/fw_printenv $(1)/usr/sbin
-	$(LN) fw_printenv $(1)/usr/sbin/fw_setenv
 	$(INSTALL_BIN) ./uboot-envtools/files/fw_printsys $(1)/usr/sbin
-	$(INSTALL_BIN) ./uboot-envtools/files/fw_setsys $(1)/usr/sbin
-	$(INSTALL_BIN) ./uboot-envtools/files/fw_loadenv $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/etc/board.d
-	$(INSTALL_DATA) ./uboot-envtools/files/fw_defaults $(1)/etc/board.d/05_fw_defaults
 	$(INSTALL_DIR) $(1)/lib
 	$(INSTALL_DATA) ./uboot-envtools/files/uboot-envtools.sh $(1)/lib
 	$(INSTALL_DIR) $(1)/etc/uci-defaults
